app/static/event/eventview.css

```diff
@@ -71,5 +67,25 @@ div.event_info_row,
 }
 
 button {
-  height: 100%;
+	height: 100%;
+}
+
+.google_maps_object {
+	width: 100%;
```

Comment:
Please set the indentation in the CSS to 2 spaces

Reply:
Sure thing, my bad.

Reply:
Still not resolved :)